api.ts

```diff
@@ -1487,7 +1518,7 @@ export function makeDynamicSyncTable<K extends string, L extends string, ParamDe
     },
   });
   const getName = wrapMetadataFunction(getNameDef);
-  const getSchema = wrapMetadataFunction(getSchemaDef);
+  const getSchema = wrapGetSchema(wrapMetadataFunction(getSchemaDef));
```

Comment:
It looks like we call `makeSyncTable()` right below this, which also does the wrapping. So seems like it's getting double-wrapped? The logic ensures the output will still be correct, but wonder if we should avoid having the wrapper called multiple times? Could just not wrap here and leave a comment that it happens later in `makeSyncTable()`.